resources/event-sources/templates/uninstall.yaml

```diff
@@ -0,0 +1,110 @@
+# Copyright 2019 The Kyma Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# About:
+# This pre-delete job removes all httpsource CRs from kyma-integration namespace
+# It does so by using a job which is executed before the helm chart is deleted
+
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: {{ .Release.Name }}-rc-delete
+  namespace: {{ .Release.Namespace }}
+  annotations:
+    # Executes on a deletion request before any resources are deleted from Kubernetes
```

Comment:
The readme on annotations is there in some resources. Do you wanna have it in the first place? It's too verbose IMO.

Reply:
I had it at the beginning of every file, but only once. I am okay with deleting it. It was helpful for me, but I understand you don't need it after you saw it the first time. Will remove